packages/assistant/src/RPA/Assistant/utils.py

```diff
@@ -0,0 +1,68 @@
+from typing import List, Tuple, Optional, Union, Any
+
+from RPA.core.types import is_list_like  # type: ignore
+from .dialog_types import Options, Element
+
+
+def to_options(
+    opts: Options, default: Optional[str] = None
+) -> Tuple[List[str], Optional[str]]:
+    """Convert keyword argument for multiple options into
+    a list of strings.
+
+    Also handles default option validation.
+    """
+    if isinstance(opts, str):
+        opts = [opt.strip() for opt in opts.split(",")]
+    elif is_list_like(opts):
+        opts = [str(opt) for opt in opts]
+    else:
+        raise ValueError(f"Unsupported options type: {opts}")
+
+    if not opts:
+        return [], None
+
+    if default is None:
+        default = opts[0]
+
+    if default not in opts:
+        raise ValueError(f"Default '{default}' is not in available options")
+
+    return opts, default
+
+
+def optional_str(val: Any) -> Union[str, None]:
```

Comment:
Nit:
```suggestion
def optional_str(val: Any) -> Optional[str]:
```
has the same effect. (so with the others below)

Reply:
done